samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -121,14 +121,20 @@
   void sink(SinkFunction<? super M> sinkFn);
 
   /**
-   * Allows sending messages in this {@link MessageStream} to an {@link OutputStream}.
+   * Allows sending messages in this {@link MessageStream} to an {@link OutputStream} and then propogates the this
```

Comment:
Make this consistent for the two methods. Also, typo: "the this".

Reply:
Typo: s/propogate/propagate

Reply:
sure